recipes/r-ndjson/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set version = '0.8.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-ndjson
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ndjson_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ndjson/ndjson_{{ version }}.tar.gz
+  sha256: 22a037dbad0ae17e0ac442f45370fa7b4c4cc23f66ef1a949fe4e910dd5b7f53
+
+build:
+  number: 0
+  skip: True  # [win]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ posix }}make
+
+  host:
+    - r-base
+    - r-rcpp
+    - r-data.table
+    - r-tibble
+
+  run:
+    - r-base
+    - r-rcpp
+    - r-data.table
+    - r-tibble
+    - zlib
+
+test:
+  commands:
+    - $R -e "library('ndjson')"
+
+about:
+  home: http://gitlab.com/hrbrmstr/ndjson
+  license: MIT
+  summary: Streaming 'JSON' ('ndjson') has one 'JSON' record per-line and many modern 'ndjson'
+    files contain large numbers of records. These constructs may not be columnar in
+    nature, but it is often useful to read in these files and "flatten" the structure
+    out to enable working with the data in an R 'data.frame'-like context. Functions
+    are provided that make it possible to read in plain 'ndjson' files or compressed
+    ('gz') 'ndjson' files and either validate the format of the records or create "flat"
+    'data.table' structures from them.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
```

```suggestion
  recipe-maintainers:
    - conda-forge/r
```
Please also add the conda-forge/r team.